scripts/check_e2e_tests_are_captured_in_ci_test.py

```diff
@@ -0,0 +1,136 @@
+# Copyright 2020 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Unit tests for scripts/check_e2e_tests_are_captured_in_ci.py."""
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+import os
+
+from core.tests import test_utils
+import python_utils
+import utils
+
+from . import check_e2e_tests_are_captured_in_ci
+
+DUMMY_TEST_SUITES = [
+'accessibility', 'additionalEditorFeatures', 'additionalPlayerFeatures',
+'topicsAndSkillsDashboard', 'users']
+
+
+def mock_read_protractor_conf_file():
+    protractor_config_file = python_utils.open_file(
+        os.path.join(
+            os.getcwd(), 'core', 'tests', 'data', 'dummy_ci_tests', 'dummy_protractor.conf.js'), 'r').read()
```

Comment:
Make os.path.join(os.getcwd(), 'core', 'tests', 'data', 'dummy_ci_tests') into a named constant so you don't need to keep repeating it. Ditto elsewhere.

Also, this line is way too long. Do you have any idea why the python lint checks are not catching this?

Reply:
Done